tests/client/amo/components/TestAddonDetail.js

```diff
@@ -441,12 +441,15 @@ describe('AddonDetail', () => {
     });
 
     it('prompts you to read many reviews', () => {
+      const i18n = getFakeI18nInst();
       const root = reviewFooterDOM({
         ratingsCount: 5,
+        i18n,
       });
       const footer =
         root.querySelector('.AddonDetail-read-reviews-footer');
       assert.equal(footer.textContent, 'Read all 5 reviews');
+      assert.ok(i18n.ngettext.calledWith('Read %(count)s review', 'Read all %(count)s reviews', 5));
```

Comment:
I think the best thing would be to add a new test for the singular phrase. I don't think it's so important to add a regression test for the fact that we are now using `i18.ngettext()` instead of `ngettext()`. It seems unlikely that we'll make that same mistake again (after that function is removed) and the way you were testing the implementation (rather than the functionality) didn't seem so useful.